### DISCUSSION TITLE
Tinder like swiping

### DISCUSSION THREAD

**Discussion by gregordimi** [nolimits4web/swiper#6392] [Q&A]
Hi everyone,

Can you point me to resources how to modify swiperjs to use it for answering questions (in tinder fashion)?

How it currently works:
- swipe right = next slide
- swipe left = previous slide
- Does not record directions

What I want to achieve:
- disable previous slide
- swipe right = next slide
- swipe left = next slide
- record direction and duration for each slide in json in some form on the page

Thank you,

**Comment by nolimits4web** [comment]
There is a `oneWayMovement` parameter https://swiperjs.com/swiper-api#param-oneWayMovement

**Comment by thegauravparmar** [comment]
Were you able to find a solution yet?
